main.js

```diff
@@ -58,13 +58,11 @@ bootstrap.Client.on('message', (receivedMessage) =>{
     if (receivedMessage.content.indexOf("https://www.spelltable.com/game/") >= 0){
         let index = receivedMessage.content.indexOf("https://www.spelltable.com/game/");
         let urlSpectate = receivedMessage.content.slice(index+32);
-        let urlPlayer = "https://www.spelltable.com/game/" + urlSpectate;
         urlSpectate = urlSpectate + "?spectate";
         const spellTableEmbed = new bootstrap.Discord.MessageEmbed()
             .setColor(bootstrap.messageColorBlue)
             .setAuthor("Looks like you're trying to play a game on Spelltable!")
```

Comment:
This line is unnecessary.  Anyone pasting the spelltable link into a chat is playing spelltable.  Let's remove this line as well.